DangoPlop/Assets/Scripts/Ball_Behavioiur.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Ball_Behavioiur : MonoBehaviour {
+    private GameObject player;
+    private Rigidbody2D rb;
+    private int thrust;
```

Comment:
let's make thrust public so that we can change it to whatever we want. Sorry I didn't mean to change them all to private.